### ISSUE TITLE
Remove mask routing

### ISSUE THREAD

**Issue by Eupham** [Eupham/KernelDev#104]
I want all references to attention masks to be purged.  By this I mean, I do not want one calculated in model.py, I do not want model.py to pass so much as a attention_mask=None to the kernel.  I do not want the kernel to use or accept mask as an argument at all for routing or as an input.  Only metadata should be passed, accepted, or used for attention behavior.
